neqo-transport/src/connection.rs

```diff
@@ -1075,6 +1090,26 @@ impl Connection {
         out
     }
 
+    fn obtain_epoch_tx_crypto_state(&mut self, epoch: Epoch) -> Res<&mut CryptoDxState> {
```

Comment:
I have a replacement for this in my patch, so as long as this works, I won't review it.